### PR TITLE
Update calendar.md

### PR DESCRIPTION
**Pull request by wade-montague**
Make permission more prominent and reduce user hunting by providing links directly to the details of how the permissions are set in managed workflow.

# Why

The documentation was too subtle in its delivery of 'add permissions', what, where, how? 

# How

Updated docs with links to the relevant documented permissions 

# Test Plan

Preview docs and follow links.

# Checklist

- [ x ] Documentation is up to date to reflect these changes (eg: https://docs.expo.io and README.md).
- [ x ] This diff will work correctly for `expo build` (eg: updated `@expo/xdl`).
- [ x ] This diff will work correctly for `expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

**Review by cruzach — CHANGES_REQUESTED** [review]
can you make the same change to the SDK 41 versioned calendar.md file?

File: docs/pages/versions/unversioned/sdk/calendar.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Provides an API for interacting with the device's system calendars, events, remi
 
 ## Configuration
 
-In managed apps, `Calendar` requires `Permissions.CALENDAR`. Interacting with reminders on iOS requires `Permissions.REMINDERS`.
+In managed apps you must add the following permissions to your app.json to use `Calendar`. Android requires [Permissions.CALENDAR](permissions#permissionscalendar) and iOS requires [Permissions.REMINDERS](permissions#permissionsreminders).
```
[the remark this answers is not in the document]
Comment (wade-montague) [reply docs/pages/versions/unversioned/sdk/calendar.md]: Thanks for the feedback, I've amended it again in both locations now taking into account your point about expo-permissions being deprecated.

File: docs/pages/versions/unversioned/sdk/calendar.md [hunk 2]
```diff
@@ -17,7 +17,9 @@ Provides an API for interacting with the device's system calendars, events, remi
 
 ## Configuration
 
-In managed apps, `Calendar` requires `Permissions.CALENDAR`. Interacting with reminders on iOS requires `Permissions.REMINDERS`.
+In managed apps you must add following permissions to your `app.json`.
```
Comment (cruzach) [inline docs/pages/versions/unversioned/sdk/calendar.md:20]:
```suggestion
You must add following permissions to your `app.json`.
```
we can configure these permissions for you in bare as well 😄

File: docs/pages/versions/unversioned/sdk/calendar.md [hunk 3]
```diff
@@ -17,7 +17,9 @@ Provides an API for interacting with the device's system calendars, events, remi
...
+- Android requires `READ_CALENDAR` & `WRITE_CALENDAR` inside the `expo.android.permissions` array.
+- IOS requires `NSRemindersUsageDescription` key added to `expo.ios.infoPlist` with a string value describing the permission request to end users.
```
Comment (cruzach) [inline docs/pages/versions/unversioned/sdk/calendar.md:22]:
```suggestion
- iOS requires the `NSRemindersUsageDescription` key be added to `expo.ios.infoPlist` with a string value describing the reason for the permission request.
```

**Review by cruzach — CHANGES_REQUESTED** [review]

File: docs/pages/versions/unversioned/sdk/calendar.md (hunk 2 above)
Comment (wade-montague) [reply docs/pages/versions/unversioned/sdk/calendar.md:20]: Good to know, I've only been working in managed work flow so don't pay too much attention to the dark side :)

**Comment by wade-montague** [comment]
Committed @cruzach 's latest suggestions and reflected those changed on the sdk-41 version of the docs also.

File: docs/pages/versions/unversioned/sdk/calendar.md [hunk 4]
```diff
@@ -17,7 +17,9 @@ Provides an API for interacting with the device's system calendars, events, remi
 
 ## Configuration
 
-In managed apps, `Calendar` requires `Permissions.CALENDAR`. Interacting with reminders on iOS requires `Permissions.REMINDERS`.
+You must add following permissions to your `app.json`.
```
Comment (cruzach) [inline docs/pages/versions/unversioned/sdk/calendar.md:20]:
```suggestion
You must add the following permissions to your `app.json`.
```

File: docs/pages/versions/v41.0.0/sdk/calendar.md [hunk 5]
```diff
@@ -17,7 +17,9 @@ Provides an API for interacting with the device's system calendars, events, remi
 
 ## Configuration
 
-In managed apps, `Calendar` requires `Permissions.CALENDAR`. Interacting with reminders on iOS requires `Permissions.REMINDERS`.
+You must add following permissions to your `app.json`.
```
Comment (cruzach) [inline docs/pages/versions/v41.0.0/sdk/calendar.md:20]:
```suggestion
You must add the following permissions to your `app.json`.
```

**Review by cruzach — APPROVED** [review]

**Pull request merged**